packages/@tinacms/cli/src/cmds/statusChecks/waitForIndexing.ts

```diff
@@ -95,18 +94,23 @@ export const waitForDB = async (ctx, next, options: { verbose?: boolean }) => {
         // Index Failed
       } else if (status === STATUS_FAILED) {
         throw new IndexFailedError(
-          `Attempting to index DB responded with status 'failed', ${error}`
+          `Attempting to index but responded with status 'failed', ${error}`
         )
 
         // Index Unknown
       } else {
-        if (options.verbose) {
-          logger.info(logText(`${statusMessage}`))
-        }
+        // TODO: Should we throw an error here?
```

Comment:
Would there be a reason not to throw here?

Reply:
I am not sure. @kldavis4 When would the indexing return "unknown" ?

Reply:
unknown because means there are no events associated with the branch

Reply:
Do you think it would make sense to throw an error if this happens?

Reply:
It means that the branch is not indexed and indexing hasn't started, so yeah, probably an error would make sense

Reply:
Makes sense, updated